### ISSUE TITLE
The sort buttons do not have meaningful description

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1930]
URL: http://clarity-ks.surge.sh/datagrid

Reproduced Steps:
1. Go to Datagrid page
2. Check “Name” property of the sort buttons such as sort of Name button. Please refer to screen shot.

Actual Results:
The “Name” property do not have meaningful description.

Expected Results:
The “Name” property should have meaningful description. Such as “Sort of Name” for the button.

Screen shot:

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/2c1cf200-d412-41b8-88cf-5ad87828665a)

NOTE: we need to know if this is a Clarity issue or implementation issue.
